package.json

```diff
@@ -49,5 +49,5 @@
       "email": "wcandillon@gmail.com"
     }
   ],
-  "dependencies": {}
+  "packageManager": "yarn@1.22.19"
```

Comment:
this is a bogus change no?
Also a bit unrelated but I am thinking of switching back to npm for the package management.

Reply:
What are your thoughts on rn skia migrating to npm?

Reply:
I played quite a lot with package managers recently and I think there is basically no significant difference between `yarn v3+` or `npm`, but both will be nice upgrade from classic yarn. 

There is also `pnpm` which is probably fastest but since there is very little JS packages anyway I doubt it will be any significant.